src/runtime_src/tools/scripts/service_bundle.sh

```diff
@@ -182,6 +204,8 @@ funcArray=(
 	vmr_logs
 	debug_logs
 	core_dumps
+	iommu_dump
+	iommu_groups
 )
```

Comment:
can you please dump the iommu dump and iommu groups into host_info as iommu_dump.log and iommu_groups.log?

Reply:
either 2 files or 1 file combined with two logs are ok